sys/fido2/ctap/transport/hid/Kconfig

```diff
@@ -4,13 +4,16 @@
 # General Public License v2.1. See the file LICENSE in the top level
 # directory for more details.
 
-menuconfig KCONFIG_USEMODULE_FIDO2_CTAP_TRANSPORT_HID
-    bool "FIDO2 CTAP TRANSPORT HID"
-    depends on USEMODULE_FIDO2_CTAP_TRANSPORT_HID
+menuconfig MODULE_FIDO2_CTAP_TRANSPORT_HID
```

Comment:
Should transports be mutually exclusive or we could have many at the same time?

Reply:
You could have multiple transports at the same time

Reply:
Ok